test/utils/RetrivalHelper.scala

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package utils
+
+import uk.gov.hmrc.auth.core.retrieve.~
+
+object RetrivalHelper {
+  implicit class RetrievalsUtil[A](val retrieval: A) extends AnyVal {
+    def `~`[B](anotherRetrieval: B): A ~ B = uk.gov.hmrc.auth.core.retrieve.~(retrieval, anotherRetrieval)
```

Comment:
You will have to explain this to me some time :)

Reply:
it avoids us having to do new ~(A, B) which is ugly and hard to read